readme.md

```diff
@@ -0,0 +1,65 @@
+First Project
+
+📗 Table of Contents
+📖 About the Project
+🛠 Built With
+Tech Stack
+Key Features
+🚀 Live Demo
+💻 Getting Started
+Setup
+Prerequisites
+Install
+Usage
+Run tests
+Deployment
+👥 Authors
+🔭 Future Features
+🤝 Contributing
+⭐️ Show your support
+🙏 Acknowledgements
+❓ FAQ
+📝 License
+📖 [Hello- microverse]
+A project that shows git and github skills, besides used of linters and gitignore file .
+
+[Hello-microverse] is a...
+
+🛠 Built With
+Html and CSS
+
+
+Client
+Server
+Database
+Key Features
+
+[html]
+[css]
+[linters]
+
+
+
+
+💻 Getting Started
+This project can be used for a first learner of github environment
+
+To get a local copy up and running, follow these steps.
+
+Prerequisites
+In order to run this project you need:
+
+Setup
+Clone this repository to your desired folder
+
+
+👥 Authors
+Erick Peñaloza
+
+
+⭐️ Show your support
+This is an effort for start a career in development world
+
+🙏 Acknowledgments
+Do not give up, continue with hard working 
+
```

Comment:
As the previous reviewer requested, please use this [format](https://github.com/microverseinc/readme-template) for your Readme file. Right now it looks like below.
<img width="917" alt="Screenshot 2022-12-03 at 2 18 35 AM" src="https://user-images.githubusercontent.com/25887253/205405886-85364976-a3ac-48d4-bf6d-2e69624e3486.png">